devel/jenkins/Portfile

```diff
@@ -114,14 +123,12 @@ startupitem.pidfile \
     manual ${jenkins_pid}
 
 notes-append "
-To configure Jenkins service, edit parameters as follows:
+To configure the Jenkins service, edit parameters as follows:
   - Read document at https://jenkins.io/doc/book/installing/initial-settings/
   - \$ sudo vi /Library/LaunchDaemons/${jenkins_plist}
-To enable the Jenkins service, use `port load`, as follows:
-  - \$ sudo port load ${name}
```

Comment:
Of note, I removed the two lines related to `port load`, as those instructions are automatically appended to the notes by MacPorts base:

```
$ port notes jenkins
--->  jenkins has the following notes:
  To configure the Jenkins service, edit parameters as follows:
    - Read document at https://jenkins.io/doc/book/installing/initial-settings/
    - $ sudo vi /Library/LaunchDaemons/org.macports.jenkins.plist
  Once the service is enabled, Jenkins will:
    - Listen by default on: http://localhost:8080
  The default Administrator password, created after port first loaded, will be available here:
    - $ sudo cat /opt/local/var/lib/jenkins/.jenkins/secrets/initialAdminPassword

  A startup item has been generated that will aid in starting jenkins with launchd. It is disabled by default. Execute the
  following command to start it, and to cause it to launch at startup:
  
      sudo port load jenkins
```